test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -4,245 +4,246 @@
 Describe "Select-String" -Tags "CI" {
     $nl = [Environment]::NewLine
     $currentDirectory = $pwd.Path
+
     Context "String actions" {
-	$testinputone = "hello","Hello","goodbye"
-	$testinputtwo = "hello","Hello"
+        $testinputone = "hello","Hello","goodbye"
+        $testinputtwo = "hello","Hello"
+
+        it "Should be called without errors" {
+            { $testinputone | Select-String -Pattern "hello" } | Should -Not -Throw
+        }
 
-	it "Should be called without errors" {
-	    { $testinputone | Select-String -Pattern "hello" } | Should -Not -Throw
-	}
+            it "Should return an array data type when multiple matches are found" {
```

Comment:
lines 16 to 19 are indented over one too many

Reply:
fixed